tests/test_pandas.py

```diff
@@ -358,8 +367,7 @@ class TestPrinting(base.BasePrintingTests):
 
 
 class TestReshaping(base.BaseReshapingTests):
```

Comment:
Can't you completely remove it?

Reply:
No, because the base class contains the extension array upstream tests which every extension array should pass. See [official pandas docs](https://pandas.pydata.org/pandas-docs/stable/development/extending.html#testing-extension-arrays). We've just skipped one or the other upstream test which was (in our opinion) broken.